Calibration/HcalAlCaRecoProducers/plugins/AlCaHBHEMuonFilter.cc

```diff
@@ -38,25 +39,44 @@
 // class declaration
 //
 
-class AlCaHBHEMuonFilter : public edm::EDFilter {
+namespace AlCaHBHEMuons {
+  struct Counters {
+    Counters() : nAll_(0), nGood_(0) {}
+    mutable std::atomic<unsigned int> nAll_, nGood_;
+    mutable std::vector<std::string>  names_;
```

Comment:
This container can not be used in a GlobalCache as a mutable since it can not safely be changed by multiple threads.